controllers/transactions.go

```diff
@@ -64,3 +65,79 @@ func MakeTransaction(w http.ResponseWriter, r *http.Request, context *ledgerCont
 		return
 	}
 }
+
+func GetTransactions(w http.ResponseWriter, r *http.Request, context *ledgerContext.AppContext) {
+	engine, err := models.NewSearchEngine(context.DB, "transactions")
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	defer r.Body.Close()
```

Comment:
Can we move this before initialising NewSearchEngine? If it returns error we would not really use the engine object.